src/main/java/uk/gov/beis/els/service/DocumentRendererService.java

```diff
@@ -66,13 +65,7 @@ public ResponseEntity processImageResponse(ProcessedInternetLabelDocument proces
   }
 
   private String generatePdfFilename(ProcessedEnergyLabelDocument processedDocument) {
-    String filename;
-    if (StringUtils.isBlank(processedDocument.getSupplierName()) || StringUtils.isBlank(processedDocument.getModelName())) {
-       filename = processedDocument.getProductMetadata().getProductFileName() + ".pdf";
-    } else {
-      filename = String.format(processedDocument.getProductMetadata().getProductFileName() + " - %s - %s.pdf",
-          processedDocument.getSupplierName(), processedDocument.getModelName());
-    }
+    String filename = processedDocument.getDocument().title()+".pdf";
```

Comment:
Went for this in the end as my attempts to move it into a util function weren't very elegant

Reply:
Yeah this seems reasonable. Could you also remove the supplier+model name from `ProcessedEnergyLabelDocument` as well? They don't seem to be used now (a benefit of doing it this way I hadn't thought of), so may as well remove them.